packages/braid-design-system/src/lib/components/Toggle/Toggle.tsx

```diff
@@ -170,6 +170,9 @@ export const Toggle = forwardRef<HTMLInputElement, ToggleProps>(
           paddingRight={
             appliedTogglePosition === 'leading' ? undefined : 'xsmall'
           }
+          display="flex"
+          flexDirection="column"
+          justifyContent="center"
```

Comment:
You could do `alignItems: center` instead and then you wouldn't need to change the flex direction. Not a huge deal, but it communicates the intent that you're trying to vertically center something that's being laid out horizontally, as opposed to spacing out something that's laid out vertically (current behaviour).

Feel like there might be an opportunity to abstract something here to ensure consistency with other inline labels like the ones used for `Checkbox` and `RadioGroup`. Not sure though. Regardless it's worth tackling separately.

Reply:
Good point. I mentally picture it about as two columns, but this is simpler

Reply:
I don't think this worked as intended, the screenshots have changed 😅

Reply:
Hmm ok, I think it's because the default value for `justifyContent` is `flexStart`, which moves all the children horizontally to the left of the container (in a row). We can add `justifyContent: flexEnd` to fix this, but now it's back to three lines anyway. Probably will just revert this change because the defaults happen to work for the case of column and it makes sense to me to think of the vertical alignment as a column